apps/recovery-portal/src/main/webapp/password-reset.jsp

```diff
@@ -163,7 +163,7 @@
                             %>
 
                            <div class="field">
-                                <label>
+                                <label for="reset-password2">
```

Comment:
Shall we name this `confirm-password`?

Reply:
adjusted with b19bce672761ac6cde9022401cc4ac469aa6fcf0